sequencer/src/persistence/fs.rs

```diff
@@ -276,22 +271,13 @@ impl Inner {
         &self,
         view: ViewNumber,
         consumer: &impl EventConsumer,
-    ) -> anyhow::Result<Vec<RangeInclusive<u64>>> {
+    ) -> anyhow::Result<Vec<RangeInclusive<ViewNumber>>> {
         // Generate a decide event for each leaf, to be processed by the event consumer. We make a
         // separate event for each leaf because it is possible we have non-consecutive leaves in our
         // storage, which would not be valid as a single decide with a single leaf chain.
         let mut leaves = BTreeMap::new();
-        for entry in fs::read_dir(self.decided_leaf_path())? {
-            let entry = entry?;
-            let path = entry.path();
-
-            let Some(file) = path.file_stem().and_then(|n| n.to_str()) else {
-                continue;
-            };
-            let Ok(v) = file.parse::<u64>() else {
-                continue;
-            };
-            if v > view.u64() {
+        for (v, path) in view_files(self.decided_leaf_path())? {
```

Comment:
More descriptive name than `v`? Looking at this PR diff, or file in general, I think it would be nice to be a bit more explicit with names that describe different kinds of view number.